packages/coding-standards/src/Sniffs/ConstantVisibilityRequiredSniff.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\CodingStandards\Sniffs;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Util\Tokens;
+
+class ConstantVisibilityRequiredSniff implements Sniff
+{
+    /**
+     * @return int[]
+     */
+    public function register(): array
+    {
+        return [\T_CONST];
+    }
+
+    /**
+     * @param \PHP_CodeSniffer\Files\File $file
+     * @param $constPosition
+     */
+    public function process(File $file, $constPosition): void
+    {
+        if (!$this->isConstInsideClass($file, $constPosition)) {
+            return;
+        }
+
+        $previousTokenEndPosition = $file->findPrevious([\T_SEMICOLON, \T_CLOSE_CURLY_BRACKET], $constPosition) ?: 0;
```

Comment:
I don't understand what expresses the variable and why it is passed into methods below. Could you please extract the line into a private method and use the new method inside of methods below so you don't have to pass the variable?